numba/types/functions.py

```diff
@@ -53,6 +53,9 @@ def format(self):
             if loc:
                 msgbuf.append('{}raised from {}'.format(indent, loc))
 
+        likely_cause = ("This error is usually caused by passing an "
+                        "unsupported argument type to the named function.")
+        msgbuf += [_termcolor.errmsg(likely_cause)]
```

Comment:
If `msgbuf` is a `list`, couldn't you just do `msgbuf.append(_termcolor.errmsg(likely_cause))`? This syntax is a bit opaque IMO.

Reply:
thanks. yes. changed.